ocs_ci/ocs/ocp.py

```diff
@@ -375,6 +375,23 @@ def new_project(self, project_name):
             return True
         return False
 
+    def delete_project(self, project_name):
+        """
+        Delete a project.  A project created by the new_project function does
+        not have a corresponding yaml file so normal resource deletion calls
+        do not work
+
+        Args:
+            project_name (str): Name of the project to be deleted
+
+        Returns:
+            bool: True in case project deletion succeeded, False otherwise
+        """
+        command = f"oc delete project {project_name}"
+        if f' "{project_name}" deleted' in run_cmd(f"{command}"):
+            return True
```

Comment:
it would be helpful if you have logs printed

Reply:
Yes.  I'm adding this.